### PR TITLE
More Changes For User Engagements (For Admins)

### PR DESCRIPTION
**Pull request by frimpongopoku**
## Works with frontend PR: https://github.com/massenergize/frontend-admin/pull/855

- [X] Modified `communities.actions.completed` to do more complex counts given (community list, action list, and/or time range)

### CODE REVIEW

File: src/_main_/settings.py [hunk 1]
```diff
@@ -27,8 +27,8 @@
 # ********  LOAD CONFIG DATA ***********#
 # DJANGO_ENV can be passed in through the makefile, with "make start env=local"
 DJANGO_ENV = os.environ.get("DJANGO_ENV","remote")
-RUN_SERVER_LOCALLY = False
-RUN_CELERY_LOCALLY = False
+RUN_SERVER_LOCALLY = True
+RUN_CELERY_LOCALLY = True
```
Comment (abdullai-t) [inline src/_main_/settings.py:31]: Is there a reason why you committed these changes?

**Review by abdullai-t — APPROVED** [review]
LGTM✅

File: src/_main_/settings.py (hunk 1 above)
Comment (BradHN1) [reply src/_main_/settings.py:31]: I'll fix it

**Review by BradHN1 — APPROVED** [review]
LGTM

**Pull request merged**